src/irr/resource/exampleResource.txt

```diff
@@ -0,0 +1 @@
+aaaaaaaaaaaaaaaaaaaaaaaaaaab
```

Comment:
make an example out of the built-in shaders (for that you need to work off the `shader_pipeline` branch)